vmdb/app/models/zone.rb

```diff
@@ -202,7 +202,7 @@ def storages
 
   def miq_proxies
     ems_ids = self.ext_management_systems.collect {|e| e.id }
-    MiqProxy.all(:include => :host, :conditions => ["hosts.ems_id in (?)", ems_ids])
+    MiqProxy.includes(:host).where("hosts.ems_id in (?)", ems_ids).to_a
```

Comment:
Should this be `.includes`, `.joins` or `.references` (is that last one even a thing? I can't remember anymore, but I remember the long talks in rails about it).  cc @tenderlove